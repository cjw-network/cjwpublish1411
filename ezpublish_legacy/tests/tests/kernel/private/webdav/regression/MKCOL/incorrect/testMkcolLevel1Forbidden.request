<?php

// Test if MKCOL returns '403 Forbidden' when creating a folder under the siteaccess

$_SERVER['REQUEST_URI'] = '/' . $GLOBALS['ezc_siteaccess'] . '/new folder';
$_SERVER['REQUEST_METHOD'] = 'MKCOL';

?>
