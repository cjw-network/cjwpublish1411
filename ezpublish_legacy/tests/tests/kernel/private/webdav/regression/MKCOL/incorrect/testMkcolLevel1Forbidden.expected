Server: eZComponents/dev/ezcWebdavTransportWrapper
Content-Type: text/plain; charset="utf-8"
Content-Length: 21
HTTP/1.1 403 Forbidden

Authorization failed.
